java-client-kubevirt.spec.in

```diff
@@ -76,7 +76,7 @@ make %{?_smp_mflags}
 # necessary because jdk 1.8 comes as default with xmvn
 export JAVA_HOME="%{_java_jdk_home}"
 
-%mvn_build -i -- %{?_mvn_opts}
+%mvn_build -i -X -d -- %{?_mvn_opts}
```

Comment:
What does -d switch does?  I cannot see that option in xmvn --help  (v3.0.0)

Reply:
Full debugging: https://docs.fedoraproject.org/en-US/java-packaging-howto/manpage_mvn_build/

Reply:
Ah right - thats from the macro itself rather than from xmvn :+1: